### DISCUSSION TITLE
Junction VIII not working with a localized version of FFVIII

### DISCUSSION THREAD

**Discussion by Khrone90** [tsunamods-codes/Junction-VIII#5] [Q&A] [answered]
As of right now the application actively seeks only for ff8_en.exe and doesn't seem to support any of the localized exes. Trying to rename a localized exe into ff8_en.exe makes JVIII to recognize it as a pirated copy. If an original ff8_en.exe (obtained by switching language in Steam) is placed in a localized installation, the game launches but crashes just before showing the ffnx logo.

I don't know if this is something that should be addressed in JVIII, or if a Game Converter (just like with 7thHeaven) is needed. Maybe it's easier to just make mods to change the language (as it is for FFVII, even if that's because of a lack of official localized versions).

**Comment by julianxhokaxhiu** [comment, marked as answer]
Thank you very much for opening this discussion, however Junction VIII supports only the English language of the game and it will never support any other language. This is to keep manageable the complexity around handling all the possible variations of languages and mods per language, which is unsustainable for the dimension of the team we currently have.

Hope this helps. Cheers and I hope you will still be able to enjoy Junction VIII!
